app/js/lib/view_handler/attribute_value_handler.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+	_ = require( 'underscore' ),
+
+	AttributeValueHandler = {
```

Comment:
The code is very generic, making `AttributeValueHandler` an appropriate name. However the constructor is very specific to the fields affected by country code. Please change either the name of the class or the constructor.